test/plugin/test_out_google_cloud.rb

```diff
@@ -70,6 +70,50 @@ def test_server_error
     assert_equal 1, exception_count
   end
 
+  def test_prometheus_metrics
```

Comment:
As I said before, the code in the non-gRPC and gRPC tests is nearly identical. Can you please add a TODO to refactor it later? Don't put a name -- we'll pick it up.